src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -713,6 +713,17 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
     }
 
     /**
+     * Create and return and un-saved model instance.
+     *
+     * @param  array  $attributes
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function make(array $attributes = [])
```

Comment:
I was just having this conversation with @calebporzio because I was accustomed to seeing `create()` as a static method on `Model.php` but it [has now been moved](https://github.com/laravel/framework/commit/bbab8abe98bd43dc3f3882194ec5c8c40d37ed21) to a dynamic method on the Eloquent Builder.

Looks like `Model`'s `__callStatic()` [method](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Eloquent/Model.php#L1336), which this is hitting, will call it dynamically.

I may be misunderstanding, but it seems safe to me.